sites/skeleton.dev/src/routes/(inner)/components/app-shell/+page.svelte

```diff
@@ -226,10 +226,11 @@
 			<CodeBlock
 				language="ts"
 				code={`
+import type { AfterNavigate } from '@sveltejs/kit';
 import { afterNavigate } from '$app/navigation';
 
-afterNavigate((params: any) => {
-    const isNewPage: boolean = params.from && params.to && params.from.route.id !== params.to.route.id;
+afterNavigate((params: AfterNavigate) => {
```

Comment:
While technically correct, this does pose a notable issue - it's SvelteKit specific. While Skeleton only provides official support for SvelteKit, we do not want to completely alienate users using Vite+Svelte or Astro.

I'm going to open this up to other core team members to decide how we want to handle this. What we might need to do is either provide multiple snippets and/or explain why you would use one approach compared to another.

Reply:
Certainly. The example already was opinionated with SvelteKit, so I simply added the correct types.

I believe tabs of examples might do well for different frameworks - though it may get out of hand eventually.

Reply:
For now we'll go forward with your changes. But I may revisit this in the future. Thanks for your contribution!